src/__test__/backend/mcode/services/CLQOutcomesService.test.js

```diff
@@ -0,0 +1,117 @@
+import {
+    expect
+} from 'chai';
+import _ from 'lodash'
+import CLQOutcomesService from '../../../../mcode-pilot/services/outcomes/CLQOutcomesService'
+import BreastMainTreatmentDiabetesHypertensionJaneV05 from '../../../../dataaccess/BreastMainTreatmentDiabetesHypertensionJaneV05.json';
+import PatientRecord from '../../../../patient/PatientRecord.jsx';
+import getProps from '../../../../mcode-pilot/utils/recordToProps'
+import '../../../../model/init.js';
+import expectedFilter from './filter.json'
+import response from './response.json'
+import rows from './rows.js'
+import processed from './processed.js';
+const nock = require('nock');
+
+
+describe("CLQOutcomesService", () => {
+    // this removes the unquie row identifiers for a set of results to allow for testing
+    // otherwise the rows are not determinable in advance and tests will fail on not matching 
+    // row ids.
+    let removeRowIds = (outcomes) => {
+        outcomes.similarPatientTreatmentsData.forEach((item) => delete item.id)
+    }
+
+    let clqService = new CLQOutcomesService({
+        serviceUrl: "http://localhost/outcomes",
```

Comment:
We seem to be wrapping our keys in quotes throughout this file - can we do the same for serviceUrl? 
```suggestion
        "serviceUrl": "http://localhost/outcomes",
```